js/apps/admin-ui/pom.xml

```diff
@@ -136,7 +136,9 @@
       "resourceUrl": "${resourceUrl}",
       "masterRealm": "${masterRealm}",
       "resourceVersion": "${resourceVersion}",
-      "isRunningAsTheme": true
+      "isRunningAsTheme": true,
```

Comment:
Let's keep this one as the last property as it is static, unlike the rest.

Reply:
Done.